pages/how_tos/bitbucket.md.erb

```diff
@@ -1,37 +1,44 @@
 # Bitbucket Repo Access
 
-- Bitbucket's build status API
-- creating or updating a PR triggers Buildkite to create a new build
-- after the build completes, the status icon in Bitbucket will update
+Connecting to Bitbucket's build status API allows repository events to trigger Buildkite builds. Events such as creating or updating a PR trigger the creation of a new build in the associated Buildkite pipeline. After the build completes in Buildkite, the status icon in Bitbucket will update to reflect the outcome. 
```

Comment:
I read this as a "updating a PR trigger" as in "what is a _pull request trigger_"